moncli/entities/column_value/complex.py

```diff
@@ -15,10 +16,30 @@ class HourValue(ColumnValue):
     pass
 
 
-class ItemLinkValue(ColumnValue):
+class ItemLinkValue(ComplexNullValue):
     """An item link column value."""
-    pass
-
+    native_type = list
+    native_default = []
+
+    def _convert(self, value):
+        list_id = value['linkedPulseIds']
```

Comment:
I came across a strange use case during testing.  

When creating an item with an item link column and querying it for the first time, the value is null.  
I then added an item in the UI.
Then I removed the value from the UI. 

Upon doing this, I was expecting the value to remain None when querying again from the API.  However, after being initially set, the API will return a value containing the 'changed_at' key but will have no 'linkedPulseIds' key, causing a KeyError on read.  

I have updated the docs accordingly.  I would simply put a try/except block looking for a KeyError and return an empty list if you trigger one.